src/styles/stars.css

```
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+
+  background-image: url(../images/star.svg);
```

please use double quotes around the URL for consistency with other URL declarations
```suggestion
  background-image: url("../images/star.svg");
```